### PR TITLE
data and analysis for PRL 130, 207001 (2023)

### PR DESCRIPTION
**Pull request by aeantipov**
This PR provides 
- Data reproducing Fig. 3 of https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.130.207001
- A notebook to analyze the data

### CODE REVIEW

**Comment by basnijholt** [comment]
Shall we add it to https://github.com/microsoft/azure-quantum-tgp/blob/b5c284272a7159b31248d0a800b2a395c5af9297/noxfile.py#L8

**Comment by aeantipov** [comment]
> Shall we add the notebook to
> 
> https://github.com/microsoft/azure-quantum-tgp/blob/b5c284272a7159b31248d0a800b2a395c5af9297/noxfile.py#L8

Done

**Comment by basnijholt** [comment]
CI seems to fail because https://github.com/mne-tools/mne-python/issues/11799

File: requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ h5netcdf
 ipython
 ipywidgets
 jinja2
-matplotlib
+matplotlib==3.7.1
```
Comment (basnijholt) [inline requirements.txt:5]: !=3.7.2 is probably better.

**Review by basnijholt — APPROVED** [review]

**Pull request merged**